test/shared-class.test.js

```diff
@@ -29,11 +29,6 @@ describe('SharedClass', function() {
       expect(sc.http.path).to.equal('/some');
     });
 
-    it('fills http.path using a normalized path', function() {
-      var sc = new SharedClass('SomeClass', SomeClass, {normalizeHttpPath: true});
```

Comment:
> Is it easy to achieve, to test for a full app scenario?

Not really. Here in strong-remoting, we usually write a test that's exercising the same call/code path as LoopBack (a full app) uses. In this particular case, it should be enough to keep this unit test that you originally proposed to remove. (I.e. revert this removal.)